examples/custom_distribution.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<p>Thinking in terms of https://github.com/pymc-devs/pymc3/issues/4899 (so it doesn't need to be done in this PR, we can add this to the issue and nothing else). pmf/pdf maybe also over/underdispersion look like terms that should go in the glossary.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Would it be helpful if I added these terms/definitions in a comment on that issue?

Reply:
That would be great yes! Thanks!

Reply:
Done!